### ISSUE TITLE
Next Regular Meetup - What should it be about?

### ISSUE THREAD

**Issue by diasdavid** [require-lx/community#83]
What about we have a Regular Meetup this month?

We've 3 talks waiting to be delivered and a Venue ready to withstand Javascript awesomeness!!

## Date
Would Friday 20th of March be ok? (so we don't clash with BREAK)

## Talks
- @joaosa https://github.com/require-lx/community/issues/31
- @jamespage - https://github.com/require-lx/community/issues/26
- @pjanuario https://github.com/require-lx/community/issues/76

@joaosa, @jamespage, @pjanuario how does this sound to you? 

## Venue
- https://github.com/require-lx/community/issues/81

@dzacarias would 20th March be a good day for Premium Minds too?

**Comment by pjanuario** [comment]
@diasdavid I am struggling with time and probably I will be like that for the next 3 months, but I am available to make an overview of ELK and to show some useful use cases and to explain how easy is to use it and take advantage of it. But I will not have time to make presentation slides. Honestly for this case I think is not an issue, and if you are ok with it, it's fine by me! ;)

**Comment by pierreozoux** [comment]
Sounds good to me!

**Comment by dzacarias** [comment]
Great!

Yeah, it works for us. 

Besides the venue and drinks, which other amenities are needed?

**Comment by dzacarias** [comment]
Are we on? ;)

**Comment by diasdavid** [comment]
Waiting on @joaosa to confirm his talk (he was almost positive ) :)

**Comment by diasdavid** [comment]
@dzacarias We are still looking for a second speaker, it seems that we will have to postpone this one

**Comment by dzacarias** [comment]
@diasdavid Oh, ok. No problem. I'll cancel our arrangements for this one. Just let us know when you have an alternate date.

**Comment by pjanuario** [comment]
@diasdavid ping me on some private message when you have news about this, since I am on rush and I can miss some notifications! :)
